src/python/turicreate/toolkits/sound_classifier/_audio_feature_extractor.py

```diff
@@ -73,39 +60,13 @@ def _preprocess_data(audio_data, verbose=True):
             print("Preprocessed {} of {} examples\n".format(len(audio_data), len(audio_data)))
         return _np.asarray(preprocessed_data), audio_data_index
 
-    @staticmethod
-    def _build_net():
-
-        net = nn.HybridSequential()
-        net.add(nn.Conv2D(channels=64, kernel_size=(3, 3), in_channels=1, padding=(1, 1), prefix='vggish_conv0_'))
-        net.add(nn.Activation('relu'))
-        net.add(nn.MaxPool2D())
-        net.add(nn.Conv2D(channels=128, kernel_size=(3, 3), in_channels=64, padding=(1, 1), prefix='vggish_conv1_'))
-        net.add(nn.Activation('relu'))
-        net.add(nn.MaxPool2D())
-        net.add(nn.Conv2D(channels=256, kernel_size=(3, 3), in_channels=128, padding=(1, 1), prefix='vggish_conv2_'))
-        net.add(nn.Activation('relu'))
-        net.add(nn.Conv2D(channels=256, kernel_size=(3, 3), in_channels=256, padding=(1, 1), prefix='vggish_conv3_'))
-        net.add(nn.Activation('relu'))
-        net.add(nn.MaxPool2D())
-        net.add(nn.Conv2D(channels=512, kernel_size=(3, 3), in_channels=256, padding=(1, 1), prefix='vggish_conv4_'))
-        net.add(nn.Activation('relu'))
-        net.add(nn.Conv2D(channels=512, kernel_size=(3, 3), in_channels=512, padding=(1, 1), prefix='vggish_conv5_'))
-        net.add(nn.Activation('relu'))
-        net.add(nn.MaxPool2D())
-        net.add(Flatten_channel_last())
-        return net
-
     def __init__(self):
         vggish_model_file = VGGish()
 
         if _mac_ver() < (10, 14):
-            # Use MXNet
-            model_path = vggish_model_file.get_model_path(format='mxnet')
-            self.vggish_model = VGGishFeatureExtractor._build_net()
-            net_params = self.vggish_model.collect_params()
-            self.ctx = _mxnet_utils.get_mxnet_context()
-            net_params.load(model_path, ctx=self.ctx)
+	    # Use TensorFlow/Keras
```

Comment:
Not a big deal but incorrectly indented comment here, I believe.

Reply:
Thanks. Fixed.